source/styles/main.css

```diff
@@ -0,0 +1,152 @@
+* {
+  font-family: "Nunito", sans-serif;
+  font-size: 14pt;
+  box-sizing: border-box;
+}
+
+.title {
+  font-size: 1.5rem;
+  font-weight: bold;
+  color: #bf4e30;
+}
+
+body {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+}
+
+main {
+  display: flex;
+  flex-grow: 1;
+  max-height: calc(100% - 4rem - 2px); /* HACK: subtract height of header */
+  padding: 1rem;
+}
+
+#job-details-list {
+  flex-grow: 1;
+  overflow-y: scroll;
```

Comment:
Can we change this to auto so the scrollbar doesnt show up when there isnt enough data to scroll

Reply:
Unfortunately this screws up the spacing between the job-details elements and the side of the screen when the scrollbar is not visible. I looked into fixing it with scrollbar styles but these are not featured enough to solve the issue, and are completely unsupported on Firefox. In my opinion, due to these issues, it is probably better for the scrollbar to stay visible at all times to provide consistent spacing.

![image](https://user-images.githubusercontent.com/97627312/204573834-a53de98b-2de3-4a84-8adb-afe88330927a.png)

![image](https://user-images.githubusercontent.com/97627312/204573904-b4a93546-ff6b-45b3-bd2d-590e827d9da3.png)

![image](https://user-images.githubusercontent.com/97627312/204574034-b6cba572-9309-493e-8658-595dceb6fcc7.png)

![image](https://user-images.githubusercontent.com/97627312/204574084-081b9f41-45e3-4f8b-8acb-3f7a4aa73a03.png)

Reply:
have you tried this? [https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-gutter](https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-gutter)

Reply:
Thanks for the suggestion! Seems to have worked. The spacing is now consistent whether or not the scrollbar is visible.

![image](https://user-images.githubusercontent.com/97627312/204652233-81421d1b-86b8-4271-80ee-286d3d8f8975.png)

![image](https://user-images.githubusercontent.com/97627312/204652259-3f773037-d18c-4f6f-a016-6727d0509ed2.png)